### DISCUSSION TITLE
希望能够支持 iOS 16 在线安装

### DISCUSSION THREAD

**Discussion by xbsite** [AlistGo/alist#1703] [Ideas]
希望能够支持 iOS 16 在线安装

**Comment by 101200** [comment]
已经支持了

**Comment by xbsite** [comment]
谢谢老板



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: 酷酷的风 ***@***.***&gt;
发送时间: 2022年9月21日 09:39
收件人: alist-org/alist ***@***.***&gt;
抄送: xbsite ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [alist-org/alist] 希望能够支持 iOS 16 在线安装 (Discussion #1703)

**Reply by xiaowang7212** [reply]
为什么我的还不行呀 3.0r1c版本

**Reply by xbsite** [reply]
文件名使用英文名@bundle-identifier.ipa

**Reply by xiaowang7212** [reply]
好的谢谢

**Reply by zhlx2835** [reply]
设置成@bundle-identifier.ipa后点击没问题，是怎么回事。安装的最新版的v3
